hadoop-hdds/common/src/main/resources/ozone-default.xml

```diff
@@ -3358,5 +3358,25 @@
     </description>
   </property>
 
+  <property>
+    <name>ozone.client.fs.listing.page.size</name>
+    <value>1024</value>
+    <tag>OZONE, CLIENT</tag>
+    <description>
+      Default listing page size value used for listing items on fs related sub-commands output. Kindly set
+      this config value responsibly to avoid high resource usage. Maximum value restricted is 5000 for
+      optimum performance.
+    </description>
+  </property>
 
+  <property>
+    <name>ozone.server.fs.listing.page.size</name>
+    <value>1024</value>
+    <tag>OZONE, OM</tag>
+    <description>
+      Default listing page size value used for listing items on fs related sub-commands output. Kindly set
+      this config value responsibly to avoid high resource usage. Maximum value restricted is 5000 for
+      optimum performance.
+    </description>
+  </property>
```

Comment:
I think name and description should reflect that it's an upper bound on page size from client, not the actual one.  (Now the "max" config from initial patch makes sense with server-side constraint.)

Reply:
> I think name and description should reflect that it's an upper bound on page size from client, not the actual one. (Now the "max" config from initial patch makes sense with server-side constraint.)

Thank you @adoroszlai for reviewing. As discussed, I have done the changes. Pls re-review.